docs/resources/node_template.md

```diff
@@ -217,6 +217,7 @@ The following attributes are exported:
 
 * `api_token` - (Required/Sensitive) Hetzner Cloud project API token (string)
 * `image` - (Optional) Hetzner Cloud server image. Default `ubuntu-18.04` (string)
+* `server_labels` - (Optional) Comma-separated list of labels which will be assigned to the server (string)
```

Comment:
The `server_labels ` argument should be a map type. Also it should be specified that this field is just available at the Hetzner Node Driver v3.7.0 and above